unittests/model_tests/neuron/test_synaptic_manager.py

```diff
@@ -406,23 +406,21 @@ def test_set_synapse_dynamics():
         # Only undelayed, all edges exist
         (range(10), [], 1000, 100, None),
         # Only delayed, all edges exist
-        ([], range(10), 1000, 100, 20),
+        ([], range(10), 1000, 100, 200),
```

Comment:
Increasing this so it's large enough to now allow a delay extension is fine; I wonder if there are any integration tests with large enough delays so that delay extensions are also triggered there.

Reply:
I know there is one that I added, but I will check the others!

Reply:
There are definitely some delay extension tests now, including one that tests various stages, so I am reasonably happy with this.